### PR TITLE
update home dashboard

### PR DESCRIPTION
**Pull request by srenaxu**
@ahsu1230 @notewio  currently, the unpublished classes do not reflect which classes are actually unpublished. i created placeholders.

### CODE REVIEW

**Comment by notewio** [comment]
you could try wrapping each section in divs and setting the margins on those, or maybe set different classes for the h3s that need different spacing

**Comment by notewio** [comment]
looks good

File: constellations/gemini-admin/src/home/home.js [hunk 1]
```diff
@@ -1,205 +1,103 @@
...
+
+                    <div class="class-section">
+                        <p class="classID">Class ID</p>
+                        <p class="edit"> </p>
+                    </div>
+                    <div class="unpublished-classes">
+                        <p class="classID">ap_calculus_spring_2020</p>
+                        <p class="edit">
+                            {" "}
+                            <button class="editB">Edit</button>{" "}
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/home/home.js:31]: I think you don't need the `{" "}`

File: constellations/gemini-admin/src/home/home.js [hunk 2]
```diff
@@ -1,205 +1,103 @@
...
+                        </p>
+                    </div>
+                    <div class="unpublished-classes">
+                        <p class="classID">ap_java_summer_2020_class_a</p>
+                        <p class="edit">
+                            {" "}
+                            <button class="editB">Edit</button>{" "}
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/home/home.js:38]: I think you don't need the {" "}

File: constellations/gemini-admin/src/home/home.js [hunk 3]
```diff
@@ -1,205 +1,103 @@
...
-        this.onClickPage = this.onClickPage.bind(this);
-        this.onClickPublishAll = this.onClickPublishAll(this);
-    }
-
-    componentDidMount() {
-        API.get("api/unpublished").then((res) => {
-            const unpublishedList = res.data;
-            this.setState({
-                programs: unpublishedList.programs,
-                classes: unpublishedList.classes,
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/home/home.js:36]: Right now looks like you're displaying fake data which is fine for "looks" purposes. The next step is to hook up actual unpublished classes to be displayed. So you're going to need this part here.

in a componentDidMount(), call API.get("api/unpublished") which is a Promise. When you receive data, setState of `classes` to `unpublishedList.classes`. You'll only need unpublished classes here.

From there, now that you `classes` is a list of objects, you should be able to render every class as a row. Reach out to people if you have questions :)

**Comment by ahsu1230** [comment]
I think some internship stuff accidentally made it into this change as well.

**Pull request closed without merging**